### DISCUSSION TITLE
Price in different currency based on the currency exchange rate instead of throwing exception

### DISCUSSION THREAD

**Discussion by morawcik** [lunarphp/lunar#1781] [Feature Requests]
It would be awesome to have option to get price in different currencies without creating new price. 
For now If I add USD I also have to add USD prices for all products, product customer groups prices and product price breaks. Ofc I can automate it with event/observer or with command but then with multiple currencies there is kida a mess with prices list especially in the 1.x when all currencies are listed default.

So having something like this would be a bless :)
```
$pricing->inCurrency('USD')->decimal(); // by currency code
$pricing->inCurrency($currency)->decimal(); // by currency model
```

**Comment by glennjacobs** [comment]
Pretty sure this is now resolved in v1, it will generate the prices based on the exchange rate when adding a new currency.
